data/metric/metrics.go

```diff
@@ -128,7 +130,7 @@ func castToFloat(value interface{}) (float64, error) {
 	return strconv.ParseFloat(fmt.Sprintf("%v", value), 64)
 }
 
-func (ms *Set) elapsedDifference(name string, absolute interface{}, sourceType SourceType) (elapsed float64, err error) {
+func (ms *Set) elapsedDifference(name string, absolute interface{}, sourceType SourceType, unsigned bool) (elapsed float64, err error) {
```

Comment:
I'd name this `unsiged` as `positive` as it's not converting to unsigned the elapsed diff, but returning errors on negative values